chain/jsonrpc-primitives/src/types/status.rs

```diff
@@ -50,7 +50,7 @@ pub struct RpcDebugStatusResponse {
 
 #[derive(Debug, serde::Serialize, serde::Deserialize)]
 #[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
-pub struct RpcHealthResponse;
+pub struct RpcHealthResponse {}
```

Comment:
This is technically a breaking change:

<img width="1077" height="726" alt="image" src="https://github.com/user-attachments/assets/a25706c2-ca9c-4480-92a2-2da0a0a4d5e6" />

Before the change any of the first two variants are valid, but with this change only the second syntax is valid.

Also, does it need `#[serde(deny_unknown_fields)]`?